src/modules/ar/ar.js

```diff
@@ -27,9 +27,10 @@ import {
 } from 'babylonjs-gui';
 
 import 'babylonjs-loaders'; // Required to load GLFT files
-import KioskAsset from '../../assets/models/SC_Kiosk.gltf';
+import KioskAsset from '../../assets/models/RP_Kiosk.gltf';
 import HelloMessage from '../../assets/audio/Hello.mp3';
 import AgentAsset from '../../assets/models/Malcolm.gltf';
+import { doc } from 'prettier';
```

Comment:
Do we need this doc import?

Reply:
It seems like we don't need this. We may have possibly added it when trying to update the WebXR button on the landing page. I commented it out, tested it, and it runs normally without it. Will remove.